src/create-menu/createMenu.js

```diff
@@ -27,48 +27,61 @@ function getMenu(userChannel, channelBoldTags) {
         },
         options: ['Option 1', 'Option 2', 'Option 3']
     };
+    let props = {
+        menuFields,
+        userChannel,
+        channelBoldTags
+    };
     let config = {
         hasDefaultQuickReply: false,
         hasWppQuickReply: true,
         isBlipImmediateMenu: false
     };
-    let menu = createMenu(userChannel, channelBoldTags, menuFields, config);
+    let menu = createMenu(props, config);
     return menu;
 }
 
 // Below are all scripts used in menu creation process
 // It should be put in the router resources in order to be used by the above script
 
 function createMenu(
-    userChannel,
-    channelBoldTags,
-    menuFields,
+    props = {
+        userChannel,
+        channelBoldTags,
+        menuFields
+    },
     config = {
         hasDefaultQuickReply: true,
         hasWppQuickReply: true,
         isBlipImmediateMenu: true
     }
 ) {
+    let { userChannel, menuFields } = props;
```

Comment:
You can remove "menuFields" from this destructuring assigment because it's not being used on the createMenu function.